playground/src/markdown/Explainer.md

```diff
@@ -76,28 +76,30 @@ You can create a program and run it. There will be no proof generated which is m
 You can step through the program and see the current VM state displayed in the Output section. 
 
 ```
-clk=0,
-op=None,
-asmop=None,
-fmp=1073741824,
-stack=[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-memory=[]
+Clock: 2001
+Stack: [1,0,4,23,1,0,2,0,23,8,50,19,17,13,11,7,5,3,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+VM Operation: And
+Assembly Instruction: "gt"
+Number of Operations: 18
+Rel. Operation Index: 11
+Memory (Addr, Mem): [0]: [2, 0, 0, 0] 
+                    [1]: [3, 0, 0, 0] 
 ```
 
-* clk is the current (clock) cycle 
-* op is the current assembler operation 
-* asmop shows shows more info on the operation
-* fmp shows the free memory pointer (fmp), which is initially set to 2^30
-* stack shows the current state ot the stack
-* memory shows (address, mem) pairs if used
+* Clock is the current (clock) cycle
+* Stack shows the current state ot the stack
+* VM Operation is the current miden vm operation (every Operation takes 1 cycle)
+* Assembly Instruction shows the current Miden assembler operation
+* Number of Operations shows cycle count for the instruction
+* Rel. Operation Index tells you at which relative cycle you are in the Instruction
```

Comment:
I believe that the assembly <-> operation relation is a bit confusing, and we should try to clarify it a bit better.

I would have the details about an operation together, since they are related. And the instruction first, since that is the thing the user will be changing, it seems to me like it is the most important item. (here and the output too)

In the docs I would also add a link to the Miden documentation. Something like this:

```suggestion
* Assembly Instruction: current Miden assembly instruction.
* Number of Operations: number of operations per instruction.
* Rel. Operation Index: current operation count.
* VM Operation: current Miden VM operation, every operation takes 1 cycle.

Additional documentation on how the VM executes its operations can be foound at the [Miden VM documentation page](https://0xpolygonmiden.github.io/miden-vm/design/programs.html)
```

Reply:
very good point, changed also the output accordingly